Social Skills.md

```diff
@@ -0,0 +1,36 @@
+#Social Skills
+ 
+Interpersonal communication is fundamental to working with people. Every individual and group has unique social characteristics related to how they communicate. Understanding the characteristics and applying them facilitates clear and effective communication. Social skills contribute to or detract from all interpersonal communication. 
+
+The skills can be broken into four broad categories. Within each category there are specific skills of particular value to software testers.
+1. Survival skills (e.g., listening, following directions, ignoring distractions, using nice or brave talk, rewarding yourself)
+•	Peer resistance skills
+2. Interpersonal skills (e.g., sharing, asking for permission, joining an activity, waiting your turn)
+•	Persuasion
```

Comment:
Replace the "dot" character with a `*` and space after it to make these real bullets.